docs/assets/scss/_sidebar.scss

```
@@ -373,9 +367,11 @@
     font-size: 1.2rem;
   }
 
-  & .td-sidebar-nav__section.without-child > a.td-sidebar-link {
-    font-weight: 400;
-    
+  // Leaf pages
+  & .td-sidebar-nav__section.without-child a.td-sidebar-link {
+    font-family: "Qanelas Soft", sans-serif;
+    font-weight: 100;
+    // background-color: green;
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)